# Cuántas veces es necesario que un médico presente el examen ENARM para que tenga al menos 50% de probabilidad de éxito

In [1]:
dataset <- read.csv("enarm.csv")

In [2]:
head(dataset)

ï..ENTIDAD.FEDERATIVA FACULTAD.O.ESCUELA.DE.MEDICINA                 
1 AGUASCALIENTES        UNIV. AUT. DE AGUASCALIENTES                   
2 AGUASCALIENTES        UNIV. CUAUHTEMOC - PLANTEL AGUASCALIENTES      
3 BAJA CALIFORNIA       CENTRO EST. UNIV. XOCHICALCO, U. ENSENADA      
4 BAJA CALIFORNIA       CENTRO EST. UNIV. XOCHICALCO, U. TIJUANA       
5 BAJA CALIFORNIA       UNIV. AUT. DE B.C. - CAMPUS VALLE DE LAS PALMAS
6 BAJA CALIFORNIA       UNIV. AUT. DE B.C., U. ENSENADA                
  SUSTENTANTES SELECCIONADOS PROMEDIO.GENERAL AGNO
1 239          84            67.52            2017
2  72           9            63.81            2017
3 247          48            61.16            2017
4 152          30            62.48            2017
5 135          16            62.70            2017
6 188          55            64.88            2017

Calculando la eficiencia

In [18]:
efi <- dataset$SELECCIONADOS/dataset$SUSTENTANTES

In [29]:
Pdeaprobar <- median(efi,na.rm=T)

In [25]:
summary(efi)

   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA's 
 0.0000  0.1429  0.2156  0.2426  0.3075  1.0000      11 

In [32]:
Intentouno <- 1-Pdeaprobar
Intentouno

[1] 0.7844365

In [38]:
Intentodos <- Intentouno*(Intentouno)
Intentodos

[1] 0.6153406

In [39]:
Intentotres <- Intentodos*(Intentouno)
Intentotres

[1] 0.4826956

In [40]:
1-Intentotres

[1] 0.5173044

La probabilidad en el intento 3 de aprobrar es mayor a .5, es decir, .517304